docs/platforms/android/profiling/index.mdx

```diff
@@ -68,3 +68,79 @@ The SDK won't run app start profiling the very first time the app runs, as the S
 The SDK will set the `isForNextAppStart` flag in `TransactionContext` if app start profiling is enabled.
 
 </Alert>
+
+## Continuous Profiling
+
+<Alert>
+
+This feature is experimental and may have bugs.
+
+</Alert>
+
+_(New in version 8.5.0)_
+
+The current profiling implementation stops the profiler automatically after 30 seconds (unless you manually stop it earlier). Naturally, this limitation makes it difficult to get full coverage of your app's execution. We now offer an experimental continuous mode, where profiling data is periodically uploaded while running, with no limit to how long the profiler may run.
```

Comment:
`m`: I find this wording sub-optimal: "The current profiling implementation stops the profiler automatically after 30 seconds" 

- to the user reading this, it's probably unclear what the "current profiling implementation" is
- I thought the main drawback was that it required transactions to run (as pointed out in the next paragraph): "Previously, profiles only ran in tandem with performance transactions that were started either automatically or manually with `Sentry.startTransaction`." - maybe shorten/merge the two paragraphs?

Reply:
We are using the same wording in all SDKs, so i'll keep it as is at the moment
I'll take this feedback to rephrase it when we GA in all SDKs, though